react/bingo/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Move to test folder

Reply:
Will work on the comment in the new project. Will resolve the discussion post incorporation of this change in the new project